### ISSUE TITLE
Laravel Reverb Message pusher.error

### ISSUE THREAD

**Issue by xerk** [laravel/reverb#136]
### Reverb Version

@beta

### Laravel Version

10.47

### PHP Version

8.2

### Description

I have installed Larave Reverb on Laravel 10  and after installing everything I websocket fetch pusher error message

`{"event":"pusher:error","data":"{\"code\":4001,\"message\":\"Application does not exist\"}"}`

### Steps To Reproduce

My Event:
```
 public function broadcastOn()
    {
            return new PresenceChannel('reverb');
   }
```

```
import Echo from 'laravel-echo';
import Pusher from 'pusher-js';
window.Pusher = Pusher;

window.Echo = new Echo({
  broadcaster: 'reverb',
  key: import.meta.env.VITE_REVERB_APP_KEY,
  wsHost: import.meta.env.VITE_REVERB_HOST,
  wsPort: import.meta.env.VITE_REVERB_PORT ?? 80,
  wssPort: import.meta.env.VITE_REVERB_PORT ?? 443,
  forceTLS: (import.meta.env.VITE_REVERB_SCHEME ?? 'https') === 'https',
  enabledTransports: ['ws', 'wss'],
});

// PresenceChannel('reverb')

window.Echo.join('reverb')
  .here((users) => {
    console.log('here', users);
  })
  .joining((user) => {
    console.log('joining', user);
  })
  .leaving((user) => {
    console.log('leaving', user);
  });
  ```

**Comment by sprtk-ches** [comment]
FWIW, I'm having the same exact issue. This is using a fresh Laravel 10 project + Herd on windows 11. Everything is boilerplate using the code from the documentation pretty much.

Also when I dispatch an event, I get the following error

`Pusher error: No matching application for ID [...]`

**Comment by mousa4007** [comment]
@xerk

**Comment by dembal1990** [comment]
+1

**Comment by Karu272** [comment]
same issue. I use laravel sanctum api setup with vue 3 as front end. stuck with 

`{event: "pusher:error", data: "{"code":4001,"message":"Application does not exist"}"}
data
: 
"{\"code\":4001,\"message\":\"Application does not exist\"}"
event
: 
"pusher:error"`

**Comment by medero** [comment]
I'm not using Echo/Pusher but in debugging I found that ws://localhost:8080/app/[app_id] which using the REVERB_APP_ID started working when [app_id] was replaced by the APP KEY - although in the configs with Echo above I see that everyone is supplying `  key: import.meta.env.VITE_REVERB_APP_KEY` so I'd think that would work...

**Comment by joedixon** [comment]
What does the request to connect to the WebSocket server look like in your network tab?

**Comment by xerk** [comment]
@mousa4007 yes I solved it, I was using wrong key make sure u are using the key of reverb

**Comment by yosha10** [comment]
> @mousa4007 how you solve this?

**Comment by paulocastellano** [comment]
> @mousa4007 yes I solved it, I was using wrong secret make sure u are using the secret of reverb

can explain better how you fixed it?

**Comment by sepgg** [comment]
Hello everyone, the problem went away when I deleted the config/reverb.php file.

**Comment by joserojasrodriguez** [comment]
Same error here, any solution?

**Comment by phpmac** [comment]
找到问题了，修改 `.env` 参数需要重新运行 `php artisan reverb:start`

**Comment by nasiriyima** [comment]
> same issue. I use laravel sanctum api setup with vue 3 as frontend. stuck with
> 
> `{event: "pusher:error", data: "{"code":4001,"message":"Application does not exist"}"} data : "{\"code\":4001,\"message\":\"Application does not exist\"}" event : "pusher:error"`

In my case I was you using the wrong REVERB_APP_KEY in laravel-echo font end

**Comment by dembal1990** [comment]
I had the same error. The problem was in the config/reverb.php file. The app definitions need to be under the apps key, which should be under another apps key.
Correct: 
```php
   'apps' => [

        'provider' => 'config',

        'apps' => [
            [
                'key' => env('REVERB_APP_KEY'),
                 ...
            ],
        ],

    ],
```

My wrong config:

```php
   'apps' => [
        'provider' => 'config',
         'key' => env('REVERB_APP_KEY'),
         ...
    ],
```

**Comment by Tschucki** [comment]
In case someone stumbles across the error again.
For me the problem was that I had 2 Laravel apps with Reverb running on the same server and logically both Reverb apps cannot listen to port 8080 at the same time.
Then try running Reverb on a **different port**.
With Laravel Forge this is super easy to change if you start the Reverb Daemon.

I noticed this when I simply tried running
```bash
php artisan reverb:start --debug
```

Also, don't forget to build your JS after changing your Reverb Credentials.
